src/pages/dashboardTab/components/tabView.js

```diff
@@ -427,7 +427,7 @@ const TabView = ({
           type="button"
           ref={saveButton}
           onClick={exportFiles}
-          className={classes.button}
+          className={[classes.button, classes.bottomBtn].join(' ')}
```

Comment:
we can use method from bento-component to combine css classes,  <div className={cn(classes.headerMainTitle, classes.marginTop23)}>

Reply:
updated